packages/kbn-optimizer/src/node/index.ts

```diff
@@ -17,8 +17,4 @@
  * under the License.
  */
 
-// register and polyfill need to happen in this
-// order and in separate files. Checkout each file
-// for a much more detailed explanation
-require('./register');
-require('./polyfill');
+export * from './node_auto_tranpilation';
```

Comment:
@spalger why don't we create a new package called `@kbn/node-env` or just `@kbn/node` in order to store that logic plus the one we have on `src/setup_node_env`? Do you think that will be to much and we should just add it to the `kbn/optimizer` extending its boundaries?

Reply:
I think it would be too little to justify a new package personally, and it fits into what I had imagined `@kbn/optimizer` would become. I also plan to move the rest of setup_node_env into `@kbn/optimizer/node` but just taking the simplest path for now.